install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -94,12 +94,12 @@ data:
       controlPlaneAuthPolicy: MUTUAL_TLS
       #
       # Address where istio Pilot service is running
-      discoveryAddress: istio-pilot.{{ .Release.Namespace }}:15005
+      discoveryAddress: istio-pilot.{{ .Release.Namespace }}:15011
```

Comment:
Please remember to update the values in istio-remote templates as well.

How does it work now in agent ?

Reply:
Checked that - there is an override that hardcodes the port ( waiting for this PR I guess). We need to remove that probably, but can be separate PR.

Reply:
However - updating istio-remote/tempalate is required.

Reply:
Thanks. In addition to the istio-remote I went over all code and replaced with the updated pilot ports.
Pilot now has only 15010 (grpc), 15011 (sgrpc), 9093 (monitoring). All the rest old/legacy are irrelevant so I thought it's better to cleaned up their occurrences to avoid confusion and copy-pasting.